src/main/java/mate/academy/MergeSortAction.java

```diff
@@ -1,8 +1,70 @@
 package mate.academy;
 
-public class MergeSortAction {
+import java.util.concurrent.RecursiveAction;
+
+public class MergeSortAction extends RecursiveAction {
+    private final int[] array;
+    private int left;
+    private int right;
```

Comment:
The fields 'left' and 'right' should be made final if they are not intended to be changed after the initial construction of the object. This would make it clear that the range they represent is fixed for the lifetime of the object.